data/webauthn-ctap2.yml

```diff
@@ -0,0 +1,22 @@
+title: Missing support for CTAP2/FIDO2 in WebAuthn
+tags:
+  - webauthn
+
+# Even though this is affecting big properties, the issue is only affecting
+# users who want to set up 2fa with a hardware token, and the assumption is that
+# this is a relatively small subset of users.
+severity: high
+user_base_impact: small
```

Comment:
FWIW I think this would suggest it's not a webcompat-priority P1, I don't know which assessment is correct, but they at least seem a little inconsistent.

Reply:
I think I would agree, and the Bugzilla bug could have been a P2. However, [there is now active work going on](https://bugzilla.mozilla.org/show_bug.cgi?id=1530370#c25) and this has been pushed to a P1 for Platform as well, so I think this is fine in this case.